coverage_comment/github.py

```diff
@@ -195,3 +195,12 @@ def create_missing_coverage_annotation(annotation_type: str, file: str, line: in
         file=file,
         line=str(line),
     )
+
+
+def append_to_environment_file(content: str, filepath: pathlib.Path):
+    with filepath.open(mode="a") as file:
+        file.write(content)
```

Comment:
Technically, this function doesn't care that this is an environment file, it only appends to a file.